### PR TITLE
fix: Fixed reappearing of login screen for brief time after login

### LINKED ISSUE #1328: Login screen reappears briefly after successful login
**Describe the bug**
Upon successful super admin login

1. the screen goes blank
2. then the login screen briefly flashes 
3. the screen goes blank
4. then the organization screen appears

**To Reproduce**
- See above

**Expected behavior**
- The login screen must not be seen again after authentication is validated

**Actual behavior**
- See above

**Screenshots**
https://github.com/PalisadoesFoundation/talawa-admin/assets/16875803/9debe957-373e-40a1-a40f-a01f20e76ee9

**Additional details**
- N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship
- https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by pateldivyesh1323**
**What kind of change does this PR introduce?**

Bugfix: Fixed reappearing of login screen for brief time after login

**Issue Number:**

Fixes #1328

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/109150688/39d68cfb-f7aa-4e92-9eb9-dcbd2f48217d

**If relevant, did you update the documentation?**

No

**Summary**

### It fixes
-  reappearing of login screen for brief time after login
- reappearing of login screen for brief time if the user was already logged in

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by aashimawadhwa** [comment]
the video is not visible on my browser can you check? @pateldivyesh1323

**Review by aashimawadhwa — APPROVED** [review]
LGTM 👍🏼

File: INSTALLATION.md [hunk 1]
```diff
@@ -25,8 +26,8 @@ This document provides instructions on how to set up and start a running instanc
   - [Debugging tests](#debugging-tests)
   - [Linting code files](#linting-code-files)
   - [Husky for Git Hooks](#husky-for-git-hooks)
-    - [pre-commit hook](#pre-commit-hook)
-    - [post-merge hook](#post-merge-hook)
+      - [pre-commit hook](#pre-commit-hook)
```
Comment (rishav-jha-mech) [inline INSTALLATION.md:29]: Why change this file ?

Comment (palisadoes) [reply INSTALLATION.md:29]: Our new commit githooks now automatically regenerate the table of contents of the markdown files in the root directory across most repositories.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**